torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2945,7 +2955,10 @@ def _summon_full_params(
         writeback: bool = True,
         rank0_only: bool = False,
         offload_to_cpu: bool = False,
+        with_grads: bool = False,
```

Comment:
maybe add this to the docstring since it is a public facing API, but document the limited support and what happens when grads are None?